### PR TITLE
Issue 356

### PR DESCRIPTION
**Pull request by radoslawroszkowiak**

### CODE REVIEW

File: apps/volontulo/tests/common.py [hunk 1]
```diff
@@ -24,144 +24,144 @@
...
-        administrator1.save()
-        administrator_profile = UserProfile.objects.create(user=administrator1)
-        administrator_profile.is_administrator = True
-        administrator_profile.save()
+        user_profile.organizations.add(organization)
+
+
+def initialize_administrator(username=None, email=None, password=None):
+    u"""Initialize administrator user."""
+    username = 'admin_user@example.com' or username
```
Comment (filipgorczynski) [inline apps/volontulo/tests/common.py]: Shouldn't be reversed: username or 'admini_user@example.com'?

Comment (radoslawroszkowiak) [inline apps/volontulo/tests/common.py]: Should be :)

File: apps/volontulo/tests/views/test_offers.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from django.test import Client
 from django.test import TestCase
 
-from apps.volontulo.models import Offer
+from apps.volontulo.models import Offer, Badge
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_offers.py]: I suggest to use separate imports. It helps if somthings broke up and throw line number.

File: apps/volontulo/tests/common.py (hunk 1 above)
Comment (radoslawroszkowiak) [inline apps/volontulo/tests/common.py]: Actually, they should just be the default arguments

**Pull request merged**